dot-net/Centroid/Config.cs

```diff
@@ -35,6 +35,12 @@ public static Config FromFile(string fileName)
             set { RawConfig[index] = value; }
         }
 
+        public object this[string index]
```

Comment:
Is `object` the right return type here?  You'd have to do some kind of type manipulation with the result, which would make it less fluent...